### PR TITLE
add gatekeeper mutation policy samples

### PR DESCRIPTION
**Pull request by ycao56**

### CODE REVIEW

File: community/README.md [hunk 1]
```diff
@@ -76,8 +76,8 @@ Policy  | Description | Prerequisites
...
+[Gatekeeper mutation set owner annotation](./CM-Configuration-Management/policy-gatekeeper-annotation-owner.yaml) | Use the Gatekeeper mutatiion policy to set owner annotation on pods. | See the [Gatekeeper](https://github.com/open-policy-agent/gatekeeper). **Note**: Gatekeeper controllers must be installed to use the gatekeeper policy. See the [Gatekeeper operator policy](./CM-Configuration-Management/policy-gatekeeper-operator.yaml). [mutatingWebhook](https://github.com/open-cluster-management/policy-collection/blob/3d6775a7ddcc007d313421c1fcc25c1fbdf28fdd/community/CM-Configuration-Management/policy-gatekeeper-operator.yaml#L111) must be enabled to use gatekeeper mutation feature.
```
Comment (dockerymick) [inline community/README.md]: I recommend the following updates:

Change:
Gatekeeper mutation set owner annotation | Use the Gatekeeper mutatiion policy to set owner annotation on pods.

TO

Gatekeeper mutation policy (owner annotation) | Use the Gatekeeper mutation policy to set the owner annotation on pods

Change:

mutatingWebhook must be enabled to use gatekeeper mutation feature. 

TO

You must enable _`mutatingWebhook`_ to use the gatekeeper mutation feature. 

I recommend that all parameters be italic monospace according to the IBM Style Guide (pg. 222)

File: community/README.md [hunk 2]
```diff
@@ -76,8 +76,8 @@ Policy  | Description | Prerequisites
...
+[Gatekeeper mutation set/update image pull policy](./CM-Configuration-Management/policy-gatekeeper-image-pull-policy.yaml) | Use the Gatekeeper mutatiion policy to set or update image pull policy on pods. | See the [Gatekeeper](https://github.com/open-policy-agent/gatekeeper). **Note**: Gatekeeper controllers must be installed to use the gatekeeper policy. See the [Gatekeeper operator policy](./CM-Configuration-Management/policy-gatekeeper-operator.yaml). [mutatingWebhook](https://github.com/open-cluster-management/policy-collection/blob/3d6775a7ddcc007d313421c1fcc25c1fbdf28fdd/community/CM-Configuration-Management/policy-gatekeeper-operator.yaml#L111) must be enabled to use gatekeeper mutation feature.
```
Comment (dockerymick) [inline community/README.md]: I recommend the following updates:

Change:

Gatekeeper mutation set/update image pull policy | Use the Gatekeeper mutatiion policy to set or update image pull policy on pods.

TO

Gatekeeper mutation policy (image pull) | Use the Gatekeeper mutation policy to set or update image pull policy on pods.

Change:

mutatingWebhook must be enabled to use gatekeeper mutation feature. 

TO

You must enable _`mutatingWebhook`_ to use the gatekeeper mutation feature.

**Review by dockerymick — COMMENTED** [review]
Made a couple of comments

File: community/README.md (hunk 1 above)
Comment (ycao56) [reply community/README.md]: done

File: community/README.md (hunk 2 above)
Comment (ycao56) [reply community/README.md]: done

**Review by dockerymick — APPROVED** [review]
The changes look good to me! I recommend adding a description prior to the content of the policy. Not required, but I think it will be beneficial for the user

**Pull request merged**